### PR TITLE
Added legal pages as well as feedback mailto link

### PR DESCRIPTION
**Pull request by defong**
Footer
image

T&C
terms and conditions publish teacher training courses gov uk

Privacy
privacy policy publish teacher training courses gov uk

Cookies

cookies publish teacher training courses gov uk

### CODE REVIEW

**Comment by fofr** [comment]
The screenshots all look the same?

File: src/ui/Views/Legal/Cookies.cshtml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    <tr>
+                        <td>seen_cookie_message</td>
+                        <td>Show cookie policy banner on first visit only</td>
+                        <td>1 month</td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+
+        <p>Other ‘Publish postgraduate teacher training courses’ cookies may include:</p>
```
Comment (fofr) [inline src/ui/Views/Legal/Cookies.cshtml]: This content doesn't fit under the `Our introductory message` heading. Should this `p` be an `h3`?

File: src/ui/Views/Legal/Cookies.cshtml [hunk 2]
```diff
@@ -91,7 +91,7 @@
         <h3 class="heading-medium">
             Other cookies
         </h3>
-        <p>‘Publish postgraduate teacher training courses’ may include:</p>
+        <p>Other ‘Publish postgraduate teacher training courses’ may include:</p>
```
Comment (fofr) [inline src/ui/Views/Legal/Cookies.cshtml]: `Other ‘Publish postgraduate teacher training courses’ may include:`
should be
`Other ‘Publish postgraduate teacher training courses’ cookies may include:`

Comment (defong) [reply src/ui/Views/Legal/Cookies.cshtml]: I need a cup of now tea

**Pull request merged**